lib/jazzy/templates/task.mustache

```diff
@@ -34,7 +34,7 @@
           <div class="declaration">
             <h4>Declaration</h4>
             <div class="Swift">
```

Comment:
This should probably be a class attr of `language` instead of `Swift` — or perhaps `class="aside language"`, which removes a special case for `.Swift` in the CSS. I tend to prefer css classes that are semantic categories, not duplicated content.